### PR TITLE
Upgrade rosa/rosa hcp cluster

### PR DESCRIPTION
**Pull request by yasun1**
[OCPQE-17904](https://issues.redhat.com/browse/OCPQE-17904)
[OCPQE-17905](https://issues.redhat.com/browse/OCPQE-17905)
[OCM-5527](https://issues.redhat.com//browse/OCM-5527)

### CODE REVIEW

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-upgrade-e2e-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-upgrade-e2e-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-upgrade-e2e-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-upgrade-e2e-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-upgrade-e2e-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-upgrade-e2e-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-upgrade-e2e-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-upgrade-e2e-f7

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-upgrade-e2e-f7

**Comment by yasun1** [comment]
@xueli181114 @yingzhanredhat Please help review.

**Comment by yasun1** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-master-rosacli-aws-rosa-hypershift-upgrade-e2e-f7 periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-ec-aws-rosa-sts-f14

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml [hunk 1]
```diff
@@ -112,6 +112,20 @@ tests:
...
     workflow: rosa-aws-sts-hypershift-private-link
+- as: aws-rosa-hypershift-upgrade-e2e-f7
+  cron: 56 11 1,9,17,24 * *
+  steps:
+    cluster_profile: aws-sd-qe
+    env:
+      CHANNEL_GROUP: candidate
+      OPENSHIFT_VERSION: 4.14.2
+      ROSA_LOGIN_ENV: staging
+      UPGRADED_TO_VERSION: "4.14"
```
Comment (xueli181114) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master__rosacli.yaml:123]: How about 4.14.x?

**Comment by yasun1** [comment]
@liangxia Please help review.

**Pull request merged**